src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -1462,26 +1512,29 @@ private void SynchronizeInternal(GraphSyncData syncData)
         {
             runnerCore.Options.IsDeltaCompile = true;
 
-            if (syncData == null)
+            using (DebugModes.IsEnabled("DumpByteCode") ? new DebugByteCodeMode(runnerCore) : null)
```

Comment:
does `using null` do anything?

Reply:
MSDN says that the compiler translates the using statement into a try catch finally with a null check in the finally block
`finally
    {
        reader?.Dispose();
    }`
    
    https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement

Reply:
Also `If a null resource is acquired, then no call to Dispose is made, and no exception is thrown`
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/statements#the-using-statement